deepseg.cc

```diff
@@ -113,6 +113,18 @@ const std::vector<std::string> labels = { "background", "aeroplane", "bicycle",
 const size_t cnum = labels.size();
 const size_t pers = std::distance(labels.begin(), std::find(labels.begin(),labels.end(),"person"));
 
+enum class ModelType {
+	BodyPix,
+	DeepLab,
+	GoogleMeetSegmentation,
+	MLKitSelfie,
+};
+
+typedef struct {
+	float mean;
+	float std;
+} meta_t;
```

Comment:
```suggestion
struct stat_data {
	float mean;
	float std;
};
```

Reply:
Hm stat as in statistics? I would prefer metadata.

Agree on removing the C-style typedefs, included for consistency with rest of code.

Reply:
Apologies for the `typedef` things - that was me with a C head on. I'll pop in a chore task to tidy up syntax for C++11 stds.

Reply:
See also my other note regarding this type. Yes, given only those names I first thought of `statistics`, but given the actual use further down a different name will likely be better. Just calling it metadata won't work though, as metadata usually refers to naming and other properties. For this the current structure doesn't look "convincing" enough to warrant that name.

Reply:
Maybe `normalization_t` as in last rev?